src/polyfill/promise.js

```diff
@@ -189,7 +189,7 @@ var Promise = (function() {
         return reject(new TypeError('Promise.all accepts an array'));
       }
 
-      var args = Array.prototype.slice.call(arr);
+      var args = Array.from(arr);
```

Comment:
It might not be good to make a polyfill depend on another polyfill, esp. given that this file comes from another repo?

Reply:
Good point!